### DISCUSSION TITLE
Welcome to Open-English-Bible Discussions!

### DISCUSSION THREAD

**Discussion by openenglishbible** [openenglishbible/Open-English-Bible#331] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Timothyjchambers** [comment]
Cool: good to be here, and looks like a good/better threaded discussion board platform....

I'm Tim Chambers, and have contributed in the past to work on mostly the smaller letters of Paul, Peter, John...

**Reply by openenglishbible** [reply]
Hi Tim - yes I think this should in the long run be a good move. Sorry about the lack of notice though! Russell

**Comment by bottleofink** [comment]
Cheers. Haven't contributed in a long time, but I have still skimmed the mailing list frequently.

As a note, don't forget to update the links to the discussion forum on https://openenglishbible.org/development/ and https://openenglishbible.org/faq/

**Reply by openenglishbible** [reply]
Thanks for the reminder - I've updated them.

**Comment by Timothyjchambers** [comment]
BTW, don't know if this helps, but this work, that appears to be a single person translation, includes this copyright:

"it is identified as the Free Bible Version, any changes are identified, and derivatives are issued with the same license as the original. For a full legal description of the license https://creativecommons.org/licenses/by-sa/4.0/legalcode"

(which if used, would require OEB to be the same license)

I cannot voucher for the quality of the translation but might be something to help with Old Testament books, and to look at for translation choices.

http://www.freebibleversion.org/info.html

**Reply by openenglishbible** [reply]
Interesting. I'll have a read through - at first glance it looks quite reasonable.

**Comment by prust** [comment]
My name is Peter Rust, I'm a senior software developer and am also working half-time to spearhead an outreach ministry at a local tech college.

I have been following the project for years and highly value the idea of a free and open-source modern Bible translation.

The Gospel according to Mark is my primary focus. One of my long-term goals is to publish the gospel according to Mark with explanatory sidebar notes and "excursus" sections -- like a study Bible but focused on just one book. 

Part of my interest in the OEB has simply been due to wanting a legal way to include the full text of a book of the Bible in the project mentioned above without needing to get permission or pay (though apparently it's not so hard to get permission for some translations, such as the NET?).

The other part of my interest is in wanting to help create a dynamic equivalent translation that doesn't run roughshod over things I believe Mark is intentionally doing in the text (like the way he repeatedly uses the phrase "the way").

But I still have further to go in my Markan studies, and I have a lot of other things vying for my time and attention, so thus far I've been content to watch the project from a distance and wait for a time when/if I'll be able to be more involved.

**Reply by openenglishbible** [reply]
Hi Peter,
Great to hear from you!

My experience with the OEB has been that its 'value add' is not just that you can publish it without permission, but that you can publish an edited version without permission. This is something which is very hard to achieve with even the most free of the commercial translations such as the NET, but is is something which you can see in operation in the highlighted uses at https://openenglishbible.org/publications/, each of which comes from the different perspectives and aims of the authors.

I think in the end the OEB will be a family of translations, which would be nice.

**Comment by watsonal** [comment]
Really a question. How do I install the Accordance file into Accordance? If possible pls email response to me at watson.allen+github@gmail.com.

**Comment by openenglishbible** [comment]
Hi Watson,
Sorry for the delay in replying!
The Accordance file is an Accordance "User Bible".

Accordance's help system has info on importing user Bibles here: http://accordancefiles2.com/helpfiles/OSX13/Default.htm#topics/05_dd/importing_bibles.htm

To get the OEB User Bible into the iOS version, you will need to import into the Mac version then sync the two, the procedure in the Accordance help system should work:
http://accordancefiles2.com/helpfiles/OSX13/Default.htm#topics/02_install_etc/sync_to_mobile_devices_mac.htm%3FTocPath%3DVersion%252013%2520Installation%252C%2520Updates%252C%2520and%2520Syncing|_____8

The Accordance help forums may also help with this:
https://forums.accordancebible.com

**Comment by watsonal** [comment]
Thanks for the info. Will give it a try.

On Wed, Nov 3, 2021 at 12:06 PM Open English Bible ***@***.***>
wrote:

> Hi Watson,
> Sorry for the delay in replying!
> The Accordance file is an Accordance "User Bible".
>
> Accordance's help system has info on importing user Bibles here:
> http://accordancefiles2.com/helpfiles/OSX13/Default.htm#topics/05_dd/importing_bibles.htm
>
> To get the OEB User Bible into the iOS version, you will need to import
> into the Mac version then sync the two, the procedure in the Accordance
> help system should work:
>
> http://accordancefiles2.com/helpfiles/OSX13/Default.htm#topics/02_install_etc/sync_to_mobile_devices_mac.htm%3FTocPath%3DVersion%252013%2520Installation%252C%2520Updates%252C%2520and%2520Syncing|_____8
> <http://accordancefiles2.com/helpfiles/OSX13/Default.htm#topics/02_install_etc/sync_to_mobile_devices_mac.htm%3FTocPath%3DVersion%252013%2520Installation%252C%2520Updates%252C%2520and%2520Syncing%7C_____8>
>
> The Accordance help forums may also help with this:
> https://forums.accordancebible.com
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/openenglishbible/Open-English-Bible/discussions/331#discussioncomment-1584316>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWHOJU3CV6FPTR3XBHLXZVTUKGB2PANCNFSM5BIID3HA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
-- 
Sent from Gmail Mobile

**Comment by autistechsolutions** [comment]
Hello all,
I am new to programming and using github. I have been looking for a project I can contribute to. I was looking for open source bible software to use while practicing my programming skills. I work mostly in python. I also happen to have a B.A. in Biblical Studies and a Master's in Theological Studies.
Peace,
Thomas

**Comment by openenglishbible** [comment]
Hi Thomas
Great to hear from you. 
Depending on your time and interest there are definitely ways for you to contribute to the OEB!
Were you more interested in the programming side?
If so I'd suggest a good starting point would be seeing if you are able to download and build the OEB sources. That would familiarise you with the way the OEB build process works, and would be helpful for me because a lot of this falls into the 'works on my machine but doesn't have much (any?) documentation' side of things!
Russell

**Comment by nvjome** [comment]
Hello,
I found this project recently and I'd love to contribute! I'm interested in both the programming and editing sides.
Is there a priority list for getting the OT books edited and released?
Nolan

**Comment by jbj99** [comment]
Hi all!
I am also interested in how I can contribute to this project. My most relevant skills would probably be transcription, checking/proofreading, formatting, etc. Basically grunt work 😂 Are there still some OT books from Moffat that need to be transcribed from archive.org?
I also have some knowledge of Greek, but very limited in the Hebrew and programming side of things. 

Kind regards,
Joel

**Reply by prust** [reply]
@jbj99: In https://github.com/openenglishbible/Open-English-Bible/discussions/348#discussioncomment-4816701 @openenglishbible pointed me toward https://archive.org/details/oldtestamentnewt0000unse which is a clearer/better source of Moffat's translation.

I've been pecking away at the Psalms in this pull request: https://github.com/openenglishbible/usfm-bibles/pull/3. I've made it to Psalm 51 and am working forward, typically transcribing one Psalm a day. I think the Psalms are slightly trickier than narrative, since there are more lines and more formatting, but it's not hard once you get the hang of it. If you're interested in jumping in and accelerating that effort, I would appreciate it. You could start at Psalm 150 and work backwards, and we could meet in the middle.

The only other OT book transcribed from Moffat is Joshua (see https://github.com/openenglishbible/usfm-bibles/tree/master/Moffat). So if you'd like to work on a book independently (i.e. other than Psalms), then I think anything other than Joshua is fair game. Back [in 2018 the priority was to do Minor Prophets first](https://openenglishbible.org/blog/2018-reboot/), but that was a long time ago, and before the pivot to Moffat, so I'm not sure if @openenglishbible has a priority order for Moffat transcriptions.

Personally, I would appreciate help with the Psalms, and I think having multiple people working together on the same thing can help with synergy and getting things across the finish line (vs. trying a bunch of separate projects simultaneously and having most of them languish), but I defer to @openenglishbible.

**Reply by jbj99** [reply]
Thanks for the quick reply @prust! I'd be happy to help out with some of the Psalms whenever I have time. 
I'm still trying to work out this GitHub thing 😂 - would the best way be for me to make a fork of usfm-bibles and then create a new branch where I can add to the psalms? (just trying to follow what you have done)

**Reply by prust** [reply]
@jbj99: The new branch, as you've done, should work fine (eventually we'll need to merge the two, but no need to worry about that now, it's easier to work on separate branches until we're ready to merge).

Discussion continued here: https://github.com/openenglishbible/Open-English-Bible/discussions/358

**Comment by axylf** [comment]
Hey all!
I am very interested in collaborating with this project. I have done translation work of my own from Koine Greek and Hebrew, and I have some software skills, in particular with React and USFM. I can also help with proof reading as well as making sure the English is natural. I also want to make the gender-neutral versions of the text flow more naturally too.

The commitment to an open source Bible translation from everyone here is amazing and I wish to help in any way I can!

Peace,
Axyl F

**Reply by openenglishbible** [reply]
Hi Axyl, great to hear from you. Your skill set sounds amazingly relevant! Development on the OEB slowed down a bit last year but I'm determined to push towards getting the OT complete. Do you have a specific area which you want to look at?

**Reply by axylf** [reply]
Hola!
I started translating Ecclesiastes as a personal project, and I suppose that I could also implement work in some OT translation. (The equivalent word for הֶבֶל I used so far was “vapid,” mainly to maintain the connection to vapor and wind.)

I also started revising 2 Samuel already and want to check it against the MT. 

Also, I want to work on adjusting the gender-neutral rendition of NT to make it more natural sounding in some instances to reflect everyday speech (i.e. “whoever” as more preferable to “the person who”). I will also check the NT against critical texts and in general work on making the language flow more naturally and do proofreading.
